tests/Microsoft.Data.Analysis.Tests/ArrowIntegrationTests.cs

```diff
@@ -60,6 +61,71 @@ public void TestArrowIntegration()
+                        nullBitmapBuffer: new ArrowBuffer.Builder<byte>().Append(0xfd).Append(0xff).Build(),
+                        length: 10,
+                        nullCount: 1,
+                        offset: 0))
+                    .Append(prependColumnNamesWith + "Column3", true, new Int32Array(
+                        valueBuffer: new ArrowBuffer.Builder<int>().AppendRange(Enumerable.Range(0, 10)).Build(),
+                        nullBitmapBuffer: new ArrowBuffer.Builder<byte>().Append(0x00).Append(0x00).Build(),
+                        length: 10,
+                        nullCount: 10,
+                        offset: 0))
+                    .Append(prependColumnNamesWith + "NullableBooleanColumn", true, new BooleanArray(
+                        valueBuffer: new ArrowBuffer.Builder<byte>().Append(0xfd).Append(0xff).Build(),
+                        nullBitmapBuffer: new ArrowBuffer.Builder<byte>().Append(0xed).Append(0xff).Build(),
+                        length: 10,
+                        nullCount: 2,
+                        offset: 0))
+                    .Append(prependColumnNamesWith + "StringDataFrameColumn", false, new StringArray.Builder().AppendRange(Enumerable.Range(0, 10).Select(x => x.ToString())).Build())
+                    .Append(prependColumnNamesWith + "DoubleColumn", false, new DoubleArray.Builder().AppendRange(Enumerable.Repeat(1.0, 10)).Build())
+                    .Append(prependColumnNamesWith + "FloatColumn", false, new FloatArray.Builder().AppendRange(Enumerable.Repeat(1.0f, 10)).Build())
+                    .Append(prependColumnNamesWith + "ShortColumn", false, new Int16Array.Builder().AppendRange(Enumerable.Repeat((short)1, 10)).Build())
+                    .Append(prependColumnNamesWith + "LongColumn", false, new Int64Array.Builder().AppendRange(Enumerable.Repeat((long)1, 10)).Build())
+                    .Append(prependColumnNamesWith + "UIntColumn", false, new UInt32Array.Builder().AppendRange(Enumerable.Repeat((uint)1, 10)).Build())
+                    .Append(prependColumnNamesWith + "UShortColumn", false, new UInt16Array.Builder().AppendRange(Enumerable.Repeat((ushort)1, 10)).Build())
+                    .Append(prependColumnNamesWith + "ULongColumn", false, new UInt64Array.Builder().AppendRange(Enumerable.Repeat((ulong)1, 10)).Build())
+                    .Append(prependColumnNamesWith + "ByteColumn", false, new Int8Array.Builder().AppendRange(Enumerable.Repeat((sbyte)1, 10)).Build())
+                    .Append(prependColumnNamesWith + "UByteColumn", false, new UInt8Array.Builder().AppendRange(Enumerable.Repeat((byte)1, 10)).Build())
+                    .Build();
+
+                return ret;
+            }
+
+            RecordBatch originalBatch = CreateRecordBatch();
+            ArrowBuffer.BitmapBuilder nullBitmapBufferBuilder = new ArrowBuffer.BitmapBuilder();
```

Comment:
(nit) (and we probably need to fix it everywhere in Arrow), we should move away from the term "null bitmap", because the values in it are `true` when the value is valid and `false` when the value is `null`. Thus the more correct term is `validity bitmap`.

See https://arrow.apache.org/docs/format/Columnar.html#validity-bitmaps